app/test_main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import pytest
+from unittest import mock
+from app.main import can_access_google_page
```

Comment:
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

`You should put a blank line between each group of imports.`